reth-extension/src/pool/pool.rs

```diff
@@ -186,21 +186,43 @@ where
         let (_, tx) = self.validate(origin, transaction).await;
         self.pool.add_transaction_and_subscribe(origin, tx)
     }
+
     /*
      * Scalaris
-     * Todo: Inject abci client to send transaction out
+     * Inject abci client to send transaction out
      */
     async fn add_transaction(
         &self,
         origin: TransactionOrigin,
         transaction: Self::Transaction,
     ) -> PoolResult<TxHash> {
-        let (_, tx) = self.validate(origin, transaction).await;
-        let mut results = self.pool.add_transactions(origin, std::iter::once(tx));
-        results
-            .pop()
-            .expect("result length is the same as the input")
+        use crate::abci::abci_ext::AbciMessage;
+        use reth_primitives::{IntoRecoveredTransaction, TransactionSigned};
+
+        let dummy_pool_result = PoolResult::Ok(TxHash::default());
```

Comment:
the usage of dummy_pool_result is initialized for a default return (2 cases) - the name itself doesn't covey the right meaning.

I think we can just use 
PoolResult::Ok(TxHash::default()) as a return value. no need to use dummy_pool_result